### PR TITLE
Add all homework tasks for lesson 3

### PR DESCRIPTION
**Pull request by h96ven**

### CODE REVIEW

File: liubarskyi/HW3/3_1.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+a = int(input('Please input number a: '))
+b = int(input('Please input number b: '))
+a = a + b
+b = a - b
+a = a - b
+print(f"Now number a equals: {a}")
+print(f"Now number b equals: {b}")
```
Comment (katemalash) [inline liubarskyi/HW3/3_1.py:7]: Good Decision

File: liubarskyi/HW3/3_2.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+bytes_number = int(input("Please, input a number in bytes: "))
+kilobytes_number = bytes_number / 1024
+megabytes_number = bytes_number / 1024 / 1024
+print(f"It is {kilobytes_number} kilobytes")
+print(f"It is {megabytes_number} megabytes")
```
Comment (katemalash) [inline liubarskyi/HW3/3_2.py:5]: Good Code

File: liubarskyi/HW3/3_3.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+width = int(input("Please, input the width of the rectangle: "))
+height = int(input("Please, input the height of the rectangle: "))
+area = width * height
+perimeter = (width + height) * 2
+print(f"The area of the rectangle equals: {area}")
+print(f"The perimeter of the rectangle equals: {perimeter}")
```
Comment (katemalash) [inline liubarskyi/HW3/3_3.py:6]: Good code

File: liubarskyi/HW3/3_4.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+from math import pi
+
```
Comment (katemalash) [inline liubarskyi/HW3/3_4.py:2]: It's recommended to use 2 empty lines after imports

File: liubarskyi/HW3/3_5.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+cathetus_1 = int(input("Please, input the length of the cathetus one: "))
+cathetus_2 = int(input("Please, input the length of the cathetus two: "))
+hypotenuse = (cathetus_1 ** 2 + cathetus_2 ** 2) ** 0.5
+print(f"The length of the hypotenuse equals: {hypotenuse}")
```
Comment (katemalash) [inline liubarskyi/HW3/3_5.py:4]: Good Work

File: liubarskyi/HW3/3_6.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+print(cats)
+print(cars)
+print(elephants)
+
+#Task two
+cats_list = ['Cheshire', 'Siam', 'Angore', 'Cheshire']
+print(cats_list)
+cats_set = set(cats_list)
+print(cats_set)
```
Comment (katemalash) [inline liubarskyi/HW3/3_6.py:20]: Good Way

**Review by katemalash — APPROVED** [review]
Good Job. Please merge this pull request

File: liubarskyi/HW3/3_1.py (hunk 1 above)
Comment (h96ven) [reply liubarskyi/HW3/3_1.py:7]: Thanks

File: liubarskyi/HW3/3_2.py (hunk 2 above)
Comment (h96ven) [reply liubarskyi/HW3/3_2.py:5]: Thank you

File: liubarskyi/HW3/3_3.py (hunk 3 above)
Comment (h96ven) [reply liubarskyi/HW3/3_3.py:6]: Thanks

File: liubarskyi/HW3/3_5.py (hunk 5 above)
Comment (h96ven) [reply liubarskyi/HW3/3_5.py:4]: Thank you

File: liubarskyi/HW3/3_6.py (hunk 6 above)
Comment (h96ven) [reply liubarskyi/HW3/3_6.py:20]: Thanks

**Pull request merged**